GameWorld/View3D/Services/ResourceLibary.cs

```diff
@@ -58,15 +58,15 @@ public void Initialize(GraphicsDevice graphicsDevice, ContentManager content)
                 CommonSpriteBatch = new SpriteBatch(_graphicsDevice);
 
                 // Load default resources
-                var mr = LoadEffect("Shaders\\Pbr\\MetalRoughness\\MetalRoughness_main", ShaderTypes.Pbs_MetalRough);
+                var mr = LoadEffect("Shaders\\Pbr\\MetalRoughness\\CA_MetalRoughness_main", ShaderTypes.Pbs_MetalRough);
```

Comment:
Please dont name the file CA, just remove the prefix as it adds nothing in this context

Reply:
I do think shader names, for games, should reflect is games it is for